src/main/java/seedu/dictionote/logic/commands/SetContactDividerPositionCommand.java

```diff
@@ -17,7 +18,7 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Set the position of contact divider.\n"
             + "Parameters: Position (must be a integer between 1 to 9 inclusively)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+            + "Example: " + COMMAND_WORD + " 3";
```

Comment:
Why the change here haha

Reply:
setting the divider at position 1 isnt ideal?

Reply:
Ah okay, makes sense. Merging